[W.I.P] Track card

mashrukh-islam opened the pull request:
### Ticket Link![TrackCard]
---------------------------------------------------


### Related Links
---------------------------------------------------


### Description
---------------------------------------------------


### Steps to Reproduce / Test
---------------------------------------------------


### GIF's
---------------------------------------------------
![TrackCard](https://user-images.githubusercontent.com/87057006/125316428-bdbdf400-e355-11eb-88ab-8c7bb7e18300.gif)

mashrukh-islam committed:
Card to display track information

A component is created to display track information. The component
is imported in the ITunesSearch Container and it takes in a track
as props. The card displays track name, artist and there is also a
simple player that provides the preview for the particular track.
The trackCard component is created using floki.

mashrukh-islam committed:
Added missing test snapshot

alichherawalla commented:
nit: use git flow to name your branches

alichherawalla commented on app/components/TrackCard/index.js:
```diff
@@ -0,0 +1,45 @@
+  return (
+    <Container data-testid="track-card">
+      <Card title={track.trackName}>
+        <p>Album : {track.collectionCensoredName}</p>
```
use react-intl, no stray strings

alichherawalla commented on app/components/TrackCard/index.js:
```diff
@@ -0,0 +1,45 @@
+        <p>Album : {track.collectionCensoredName}</p>
+        <p>Perfomed By : {track.artistName}</p>
+        {track.artworkUrl100 ? <img src={track.artworkUrl100} /> : null}
+        {track.previewUrl ? <ReactPlayer url={track.previewUrl} controls={true} /> : null}
```
use `If`

alichherawalla commented on app/components/TrackCard/index.js:
```diff
@@ -0,0 +1,45 @@
+      <Card title={track.trackName}>
+        <p>Album : {track.collectionCensoredName}</p>
+        <p>Perfomed By : {track.artistName}</p>
+        {track.artworkUrl100 ? <img src={track.artworkUrl100} /> : null}
```
use `If`

mashrukh-islam committed:
Replaced stray strings with react-intl and used If component

alichherawalla commented on app/components/TrackCard/index.js:
```diff
@@ -0,0 +1,87 @@
+            <T id="performed_by" values={{ artistName }} />
+          </Col>
+          <Col size={0.5}>
+            <p>
```
lets use this instead: https://ant.design/components/statistic/

mashrukh-islam committed:
Statistic component and avatar to display data

Used statistic component to display price and release date.
Avatar is used to display the album cover

mashrukh-islam committed:
Fixed symmetry and changed location of elements
